src/js/cards.js

```diff
@@ -1,55 +1,101 @@
-export const DRAW_MS = 10000;
+// Each creature attacks at a certain rate.
+// This is the default rate in milliseconds.
 export const DEFAULT_ATTACK_RATE = 3000;
-export const STARTING_HAND_SIZE = 3;
 
+/* 
+   CARDS maps names of cards to properties/definitions.
+
+   Cards must have these properties: 
+     cost           - INT            (>=0)
+     permanent      - BOOL           stays in play when used
+
+   Cards may have these optional properties:
+     attack         - INT            (>=0)
+     defense        - INT            (>=1)
+     attackRate     - INT            (millis)
```

Comment:
i like how the name of "attackRate" is parallel to the constant's name DEFAULT_ATTACK_RATE